plugins/catalog-backend-module-ldap/api-report.md

```diff
@@ -106,17 +107,21 @@ export class LdapOrgEntityProvider implements EntityProvider {
   // (undocumented)
   static fromConfig(
     configRoot: Config,
-    options: {
-      id: string;
-      target: string;
-      userTransformer?: UserTransformer;
-      groupTransformer?: GroupTransformer;
-      logger: Logger;
-    },
+    options: LdapOrgEntityProviderOptions,
   ): LdapOrgEntityProvider;
   // (undocumented)
   getProviderName(): string;
-  read(): Promise<void>;
+  read(options?: { logger?: Logger }): Promise<void>;
+}
+
+// @public
+export interface LdapOrgEntityProviderOptions {
+  groupTransformer?: GroupTransformer;
+  id: string;
+  logger: Logger;
+  schedule: 'manual' | TaskRunner;
```

Comment:
Guess we should consider a rename of `schedule` here, although it kinda fits as is too

Reply:
Yeah I thought about that as well - it kinda fits this way as you say, turned it back and forth a bit but couldn't come up with a better fit at this point